crates/biome_css_analyze/tests/specs/nursery/noUnknownSelectorPseudoElement/invalid.css

```diff
@@ -0,0 +1,7 @@
+a::pseudo { }
+a::Pseudo { }
+a::pSeUdO { }
+a::PSEUDO { }
+a::element { }
+a:hover::element { }
+a,\nb > .foo::error { }
```

Comment:
```suggestion
a,
b > .foo::error { }
```

Reply:
I addressed it in the following commit.
48b651f48e469de3a911c1a3cd9095b5d2ffd8b1